PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/ClickToMove.rbxmx

```diff
@@ -98,7 +112,7 @@ do
 	end
 
 	local function clamp(low, high, num)
-		return math.max(math.min(high, num), low)
```

Comment:
Yeah, it's quite a bit faster. I got some benchmark information here: 
http://devforum.roblox.com/t/just-a-quick-clamp-function/31352/9?u=thegamer101